src/components/Home.css

```diff
@@ -1,3 +1,291 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700&family=Roboto+Slab:wght@100;300;400;700&display=swap');
+
```

Comment:
Good job with the general design, however, some of the elements do not match the Zeplin.

Look:

![image](https://user-images.githubusercontent.com/57421823/161352514-4df649df-0f14-460d-b8c3-61b0cf221968.png)

Therefore:

- [ ] Please, align the booklist section with the form section, so the alignment matches.  Please make sure that the big horizontal line between these 2 sections matches the alignment too.
- [ ] According to the Zeplin, the form section does not feature borders.  Please, remove them so your project looks like the zeplin.
- [ ] The category input and the "add button" are being sent to the next line.  Please, kindly resize your "title" and "author" inputs so they are shorter and make sure the whole form (all the inputs and the submit button) fits and renders in just one line.  This is an example:
![image](https://user-images.githubusercontent.com/57421823/161352969-706e5659-323a-4b4d-b71f-0d308b53190b.png)
- [ ] The borders featured on the page are the wrong width.  (You mostly make them 2px, they should be just 1 px).  Please, make them the right width.

If you implement these changes, your project will look more like the Zeplin! 💪